spec/presenters/content_embed_presenter_spec.rb

```diff
@@ -73,10 +73,10 @@
           let(:details) { { body: "some string with a reference: {{embed:contact:#{embedded_content_id}}} {{embed:contact:#{fake_content_id}}}" } }
 
           it "alerts Sentry and returns the expected content" do
```

Comment:
keeping the test wording with Sentry in it as this describes our intention?

Reply:
Yeah, that makes sense, that's what `GovukError` does anyway